tests/test-frameworks/isolation/test

```diff
@@ -13,3 +13,7 @@ bazel build :classloader-shared
 ! bazel test :classloader-shared
 
 bazel test :process :process-2_11 :process-jvm
+
+bazel test --test_arg=--framework_args=-showtimes :none | grep 'be 2.*ms'
```

Comment:
Also, a comment above this block like `# verify framework args work for each level of isolation` would be great. Thanks!